internal/plugin/client.go

```diff
@@ -106,57 +112,74 @@ func NewAccountClient(ctx context.Context, config *configEntry) (*accountClientC
 }
 
 func (clientCtx *projectClientContext) GenerateToken(projectName string, projectRoleName string, expiresIn time.Duration) (*projectToken, error) {
-	id := uuid.New().String()
-	createTokenRequest := &project.ProjectTokenCreateRequest{
-		Project:   projectName,
-		Role:      projectRoleName,
-		ExpiresIn: toDurationSeconds(expiresIn),
-		Id:        id,
-	}
+	retries := 0
+	var response *project.ProjectTokenResponse
+	var err error
+
+	for retries < totalRetries {
```

Comment:
Do we need to have a separate variable for number of totalRetries, will this always be the same as `range(retryWaitSeconds)`?